amazon-msk/cdk/lib/IotIngestAndControlStack.ts

```diff
@@ -72,7 +72,7 @@ interface IotIngestAndControlContext {
 }
 
 export class IotIngestAndControlStack extends cdk.Stack {
-  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+  constructor(scope: Construct, id: string, freeTrial: boolean, props?: cdk.StackProps) {
```

Comment:
I think the idiomatic way to do this is to have a strongly typed subtype of `cdk.StackProps` and set the `freeTrial` property of that, rather than adding it as a separate parameter to the constructor.